tests/integration/test_storage_meilisearch/test.py

```diff
@@ -374,16 +383,18 @@ def test_insert_secure(started_cluster):
     big_table = client.index("big_table")
 
     node = started_cluster.instances["meili"]
+    node.query("DROP TABLE IF EXISTS new_table")
     node.query(
         "CREATE TABLE new_table(id UInt64, data String) ENGINE = MeiliSearch('http://meili_secure:7700', 'new_table', 'password')"
     )
 
     node.query(
         "INSERT INTO new_table (id, data) VALUES (1, '1') (2, '2') (3, '3') (4, '4') (5, '5') (6, '6') (7, '7')"
     )
-    sleep(1)
+    sleep(5)
```

Comment:
why are we increasing the sleep time ?

Reply:
because 1 is not enough

Reply:
are we waiting for some operation to complete ? so executing INSERT statement is not enough for next checks of meili client? Trying to understand how having more sleep time would help

Reply:
it is specific to Meilisearch, it does several operations asynchronously, including updating documents in the index, which is checked in this test.

Reply:
https://docs.meilisearch.com/learn/advanced/asynchronous_operations.html#which-operations-are-asynchronous